.gitignore

```diff
@@ -1,3 +1,7 @@
-*.wpr
-*.wpu
-.*pyc
\ No newline at end of file
+# Ignore python byte code.
+*.pyc
+*~
+
+# Ignore editor configurations.
+.project
+.pydevproject
```

Comment:
Could you leave the `*.wpr` and `*.wpu` extensions? I still use the WingWare IDE from time to time.